garrysmod/gamemodes/terrortown/gamemode/util.lua

```diff
@@ -1,11 +1,12 @@
--- Random stuff
-
 if not util then return end
 
+local util = util
 local math = math
 local string = string
 local table = table
 local pairs = pairs
+local table = table
+local band = bit.band

```

Comment:
I'm not sure if it actually has any efficiency benefits, but they're all over TTT, so I won't touch them unless all of them are changed